### PR TITLE
Fixed bug with actor system being created and not stopped if exception occurs during validation of command line arguments BA-4061

### PR DESCRIPTION
**Pull request by grsterin**
…n occurs during validation of command line arguments [BA-4061]

[BA-4061]: https://broadworkbench.atlassian.net/browse/BA-4061

### CODE REVIEW

**Comment by cjllanwarne** [comment]
Just an FYI - if you can put the `BA-4061` in the PR title, that will let Jira automatically move the ticket to "in review" and "closed" as soon as the PR is created/merged

**Comment by cjllanwarne** [comment]
Actually... @aednichols has some "run a single workflow mode" tests - it might be nice to add this situation to those so we can avoid any regressions here. Does that sounds feasible to you Adam (or is it a much bigger change)?

**Comment by aednichols** [comment]
The single workflow runner test seems like a good place to test this, indeed.

It lives at `src/ci/bin/testSingleWorkflowRunner.sh`

Following the examples of the other test cases in that file, it should be possible to assert the tool emits the expected output under the conditions described in the ticket.

**Comment by mcovarr** [comment]
The `singleWorkflowRunner` and `dockerDeadlock` builds both failed on the last PR run. I was seeing weird errors with `dockerDeadlock` on my builds yesterday that I eventually got past by restarting the builds, but the `singleWorkflowRunner` errors look more suspicious to me.

**Comment by grsterin** [comment]
> The `singleWorkflowRunner` and `dockerDeadlock` sub-builds both failed on the last PR run. I was seeing weird errors with `dockerDeadlock` on my builds yesterday that I eventually got past by restarting the builds, but the `singleWorkflowRunner` errors look more suspicious to me.

The problem was caused by the fact that singleWorkflowRunner tests rely on application's log messages for validation and the first fix attempt broke logging: `CromwellEntryPoint.buildCromwellSystem` was calling `initLogging` method  to tamper with system properties before logback initialization. Then I moved `validateRunArguments` call to happen before the `buildCromwellSubsystem`, but turned out that `validateRunArguments` triggered logback initialization before system properties have been modified, thus making logback misconfigured.

File: server/src/main/scala/cromwell/CromwellEntryPoint.scala [hunk 1]
```diff
@@ -56,10 +58,13 @@ object CromwellEntryPoint extends GracefulStopSupport {
     * Run a single workflow using the successfully parsed but as yet not validated arguments.
     */
   def runSingle(args: CommandLineArguments): Unit = {
+    initLogging(Run)
```
[the remark this answers is not in the document]
Comment (aednichols) [reply server/src/main/scala/cromwell/CromwellEntryPoint.scala]: Never mind, I see you explained above

**Pull request merged**